### PR TITLE
[W5.3b][W11-A1]Lai Qi Wei

### PR DESCRIPTION
**Pull request by qwlai**
Ready for review.

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a person's contact.
+ */
+public class Contact {
+
+    protected final String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:8]: Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        this.isPrivate = isPrivate;
+    }
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public int hashCode() {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:22]: In general, `equals()` and `hashCode()` methods should be overridden together. Find out why!

**Review by okkhoy — COMMENTED** [review]
You can try to move the validation logic as well to the superclass.

**Comment by okkhoy** [comment]
@qwlai 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Contact.java (hunk 1 above)
Comment (qwlai) [reply src/seedu/addressbook/data/person/Contact.java:8]: Okay, will take note when assigning access modifiers to variable in future.

**Comment by qwlai** [comment]
Thank you for the comments. I have learnt something new today by looking up why is it important to override equals and hashCode together.

**Pull request closed without merging**